quintilesims/layer0
# 498 Implement dynamic environments

**What does this pull request do?**
Adds new environment_type attribute to environments where the value can be either dynamic (default) or static.


**How should this be tested?**
CLI: 
* Create environment with and without `--scale` flag to create static & dynamic environments
* Run environment list and get commands to ensure the environment output is as expected
* Run `set-scale` on dynamic and static environments and ensure output is as expected

Terraform:
* Create environment with and without `environment_type` attribute (value: 'static' or 'dynamic' [default])
* Update scale flag for both environment types
* Update environment_type 


**Checklist**


closes #498
